packages/lesswrong/components/linkPreview/PostLinkPreview.tsx

```diff
@@ -762,7 +764,7 @@ const NeuronpediaPreview = ({classes, href, id, children}: {
         </a>
 
         <LWPopper open={hover} anchorEl={anchorEl} placement="bottom-start">
-          <iframe className={classes.iframeStyling} src={url} />
+          <iframe className={classes.iframeStyling} scrolling="no" src={url} />
```

Comment:
FYI `scrolling` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#scrolling)